BlockchainTests/GeneralStateTests/stBadOpcode/badOpcodes_d101g0v0.json

```diff
@@ -95,11 +96,12 @@
     },
     "badOpcodes_d101g0v0_EIP158" : {
         "_info" : {
+            "binaryenVersion" : "BINARYEN_VERSION",
             "comment" : "",
```

Comment:
Wtf is binaryen? @axic

Reply:
It's a wasm interpreter.  Only relevant for `stEwasmTests` but somehow made it into this repository by accident.